airflow/task/task_runner/standard_task_runner.py

```diff
@@ -82,9 +82,10 @@ def _start_by_fork(self):  # pylint: disable=inconsistent-return-statements
             setproctitle(proc_title.format(args))
 
             try:
-                args.func(args, dag=self.dag)
+                args.func(args, dag=None)
```

Comment:
```suggestion
                args.func(args)
```

Also should we remove `self.dag` from constructor? It seems to be added in the same change #7090

Reply:
yes